aii-core/src/main/perl/aii-web.cgi

```diff
@@ -79,9 +82,14 @@ sub Initialize {
   # find all profiles in directory
   my $d;
 
-
-  push @profiles,map { s/\.xml$//; $_.=".".$server_domain; } sort(grep(/\.xml$/, readdir(DIR)));    
-
+  # read GET and read profiles by extension (GET ?json)
+  if ($ENV{'REQUEST_METHOD'} eq "GET" and $ENV{'QUERY_STRING'}=='json') {
```

Comment:
should be `$ENV{'REQUEST_METHOD'} and $ENV{'REQUEST_METHOD'} eq "GET" and $ENV{'QUERY_STRING'} and $ENV{'QUERY_STRING'} eq 'json'` to avoid isues with undefined ENV variables